frontend/src/concepts/trustyai/context/TrustyAIContext.tsx

```diff
@@ -72,23 +63,23 @@ export const TrustyAIContextProvider: React.FC<TrustyAIContextProviderProps> = (
   const contextValue = React.useMemo(
     () => ({
       namespace,
-      hasCR: !!explainabilityNamespaceCR,
+      hasCR: !!trustyNamespaceCR,
       crInitializing: !crLoaded,
       serverTimedOut,
       ignoreTimedOut,
-      serviceLoadError,
+      crLoadError,
```

Comment:
Well this is interesting. There's no type error wrt the context prop type `TrustyAIContextProps`. Nor is there an issue in `head` on `main` branch. The type context type still includes `serviceLoadError`.

Reply:
Yeah this is weird... even if I explicitly type the memo it doesn't complain hmmm. Seems TS isn't picking up on the indirection here. I tested this by assigning the props inline to the value prop and suddenly it's complaining. If you inline the memo TS stops complaining again....

Reply:
Bug for main: https://issues.redhat.com/browse/RHOAIENG-7251